WolvenKit.Common/Services/HashService.cs

```diff
@@ -99,23 +94,10 @@ public string Get(ulong key)
 
             // load additional
             LoadAdditional();
-            if (_additionalhashes.ContainsKey(key))
-            {
-                return _additionalhashes[key].ToString();
-            }
-
-            return "";
+            return _additionalhashes.ContainsKey(key) ? _additionalhashes[key].ToString() : "";
```

Comment:
```suggestion
            return _additionalhashes.ContainsKey(key) ? _additionalhashes[key].ToString() : string.Empty;
```

Again, I would prefer the old `if-else`.